modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,176 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
+
+For the Yomi source code, see https://github.com/openSUSE/yomi.
```

Comment:
```suggestion
```
I think this reference is not needed here.  If you think it's important enough, mention it either at the very beginning of this section (right after the intro para) or at the end.